userdocs/profiles.dev/src/components/HomepageFeatures.js

```diff
@@ -4,29 +4,32 @@ import styles from './HomepageFeatures.module.css';
 
 const FeatureList = [
   {
-    title: 'GitOps Native package manager',
+    title: 'What are Profiles?',
     Svg: require('../../static/img/undraw_docusaurus_mountain.svg').default,
     description: (
       <>
-      some cool text here
+      Profiles is a GitOps Native Package Manager. 
```

Comment:
Yeh this is what i was struggling with around how we have named things. We don't have an actual "thing" called Profiles. Profiles is just the concept, and pctl is the actual tool which manages profiles (lowercase p) which are packages.

Reply:
Also we have `What are Profiles?` (plural) then `Profiles is a..` (singular), which sort of illustrates what I am saying.

Profiles in the plural is more than one profile/package; `profiles are k8s packages` or whatever.

Profiles as a title in the singular is the concept: `Profiles is a GitOps package management mechanism` iyswim?

Reply:
changed to make it consistent everywhere with your suggestion